### ISSUE TITLE
[BUG - Release]

### ISSUE THREAD

**Issue by Sopitive** [UE4SS-RE/RE-UE4SS#594]
**Branch or Release**
3.0.1

**Game and Engine Version**
Halo The Master Chief Collection 4.21

**Describe the bug**
Crash after intro screen. With or without mods installed. I used custom signatures from a different issue that allowed things to actually work, but resulted in a crash right after the intro plays.

**Mods directory**
Default mods

**To Reproduce**
1. Launch game
2. Wait for intro screen to finish
3. Fatal Error

**Expected behavior**
Game launches and lets me into the main menu

**Desktop (please complete the following information):**
 - OS:Windows 11 Pro

[UE4SS.log](https://github.com/user-attachments/files/16215405/UE4SS.log)

**Comment by Buckminsterfullerene02** [comment]
We don't provide support for games with online PvP.

**Comment by Sopitive** [comment]
Uh... OK? The game literally has mod tools released by the game developers and an instance dedicated to turn off the anti cheat for mods, including Steam Workshop mods. If that doesn't give the green light for any kind of modification of the game then I don't know what does. The only thing the mod tools don't cover is the menus which use UE4.

**Comment by Buckminsterfullerene02** [comment]
Apologies, that is a misjudgement on my part, I see the microsoft modding tools for this game now.

**Comment by Connor-jt** [comment]
@Sopitive did you ever get this working? i recall you playing around with the UI stuff on one occasion, would be useful for something im working on

**Comment by Sopitive** [comment]
> [@Sopitive](https://github.com/Sopitive) did you ever get this working? i recall you playing around with the UI stuff on one occasion, would be useful for something im working on

No, I hadn't come back to this, but I have played around with the UI. I never fully got this working, somewhat due to the game crashing and me probably not fully understanding the tool itself.
